### PR TITLE
DOCS-752: Fix trailing slashes for SEO

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/tutorials/get-started/make-an-led-blink-with-a-raspberry-pi-and-sdk.md [hunk 1]
```diff
@@ -298,7 +298,7 @@ If you have any issues whatsoever getting the Viam SDK set up or getting your co
 
 In this tutorial, you learned the basics of controlling your robot using the Viam SDK by writing a short program in either Go or Python that makes an LED on your Raspberry Pi blink on and off.
 
-If you are looking for some projects that would be a great next step in your journey of learning about how to build robots, check out one of following [Tutorial List](..).
+If you are looking for some projects that would be a great next step in your journey of learning about how to build robots, check out one of following tutorials](../).
```
Comment (andf-viam) [inline docs/tutorials/get-started/make-an-led-blink-with-a-raspberry-pi-and-sdk.md:301]:
```suggestion
If you are looking for some projects that would be a great next step in your journey of learning about how to build robots, check out one of following [tutorials](../).
```

Comment (andf-viam) [reply docs/tutorials/get-started/make-an-led-blink-with-a-raspberry-pi-and-sdk.md:301]: Though this somehow [builds fine](https://docs-test.viam.dev/334d8f2957a4c176fbed8c46fe581fff7d6ad562/public/tutorials/get-started/make-an-led-blink-with-the-viam-app/#summary)?

**Review by andf-viam — APPROVED** [review]
LGTM, wow! Caught one typo, otherwise amazing.

File: docs/tutorials/get-started/make-an-led-blink-with-a-raspberry-pi-and-sdk.md (hunk 1 above)
Comment (npentrel) [reply docs/tutorials/get-started/make-an-led-blink-with-a-raspberry-pi-and-sdk.md:301]: weird but I guess ok

**Pull request merged**